exporter/exporterhelper/queue_batch.go

```diff
@@ -14,23 +14,13 @@ import (
 // Deprecated: [v0.123.0] use QueueBatchConfig.
 type QueueConfig = QueueBatchConfig
```

Comment:
```suggestion
```
Should this be removed as well?

Reply:
I see now that this PR depends on https://github.com/open-telemetry/opentelemetry-collector/pull/13338. I think we can address this there.